cli/docs/cli.json

```
@@ -593,6 +593,11 @@
           "name": "download",
           "about": "Downloads a support bundle",
           "args": [
+            {
+              "long": "chunk-size",
+              "short": "c",
+              "help": "The size of each range request to use when downloading bundles"
```

We should mention the unit

```suggestion
              "help": "The size in MiB of each range request to use when downloading bundles"
```

---

The units are bytes, not MiB:

```
-c, --chunk-size <CHUNK_SIZE>  The size of each range request to use when downloading bundles [default: 104857600]
```

---

Ah right, sorry about that, disregard.

---

Clarified with "in bytes"